README.md

```diff
@@ -7,16 +7,17 @@ The Bosch Connected Devices Development Kit (CDDK) has been originally used to d
 Kiso (Japanese for "basis") is an Eclipse Foundation project that is using the original code base of CDDK. Kiso is transferring the Bosch proprietary licence code piece-by-piece to EPLv2 open source. Kiso's quality goals - understandability, usability, product quality, extensibility and performance - are the key factors that enable a fast prototype and product development. By this, it reduces the time to market together with minimizing technical debts for almost all kinds of IoT "Things".
 
 ## Quick start ##
-Please take a look at the [quick start](http://kiso.rempler.de:1313/3.-user-guide/quick_start/) section in the user manual documentation.
+Please take a look at the [quick start](http://kiso.rempler.de:1313/3.-user-guide/quick_start.html) section in the user manual documentation.
```

Comment:
we should merge #123 first as this will change the structure again. => removing the number prefix

Reply:
Na, I don't think so - you never know when the other pull request is going to get reviewed and merged. What of the AE Zuender article is released in-between? We would then have a broken readme for some time (even longer than right now) - I propose "der Tag die Sorge" here :-)